### DISCUSSION TITLE
Homebrew Formulae

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#2403] [Everyday usage] [answered]
Is searchable [Homebrew Formulae](https://formulae.brew.sh/) automatically updated whenever something is change in git or it require some manual updates?

Since I am almost sure that it's automatic process I am not sure why search can't find `avtouchbar` yet. Maybe it need some time for indexing?

**Comment by SMillerDev** [comment, marked as answer]
It's updated every hour

**Reply by viking1304** [reply]
> It's updated every hour

In that case there is some issue with search. Avtouchbar formula is merged about 16 hours ago, but it still can't be found.

```
Latest commit 051de15 16 hours ago
```
[https://formulae.brew.sh/cask/avtouchbar](https://formulae.brew.sh/cask/avtouchbar)

Whatever you type, many other things will be found but not `avtouchbar`:

```
avt
avto
avtouch
touchbar
avtouchbar
bar
```

**Reply by viking1304** [reply]
Now it's fine. I guess it just needed some time after all as I assumed.
